app/urls.py

```diff
@@ -57,5 +58,15 @@
         'users/<uuid:pk>/',
         UserDetail.as_view(),
         name='user_detail'
-    )
+    ),
+    path(
+        'interests/',
```

Comment:
https://github.com/mentorpaired/core/issues/41#issue-598318640 please use the urlpatterns suggested here instead.